x/slashing/spec/README.md

```diff
@@ -1,11 +1,26 @@
-<!--
-order: 20
-title: Slashing Overview
-parent:
-  title: "slashing"
--->
```

Comment:
Do not remove these please, they are needed for the docs website

Reply:
Got it. Also added back evidence and bank ones.